flexmeasures_openweathermap/utils/modeling.py

```diff
@@ -19,6 +19,16 @@ def get_or_create_owm_data_source() -> DataSource:
     )
 
 
+def get_or_create_owm_data_source_for_derived_data() -> DataSource:
+    owm_source_name = current_app.config.get(
+        "OPENWEATHERMAP_DATA_SOURCE_NAME", DEFAULT_DATA_SOURCE_NAME
+    )
+    return get_or_create_source(
```

Comment:
`get_or_create_source` should be imported from `flexmeasures.data.queries` as of flexmeasures==0.9.